python-lib/dku_io_utils/partitions_handling.py

```diff
@@ -24,16 +24,18 @@ def get_partition_root(dataset):
     return file_path
 
 
-def get_folder_partition_root(folder):
+def get_folder_partition_root(folder, is_source=False):
```

Comment:
I am not sure the term "source" is the most adapted. This term is never used in the DSS terminology. I think `is_input` would be more self-explanatory.